tests/e2e/configuration.py

```diff
@@ -6,7 +6,7 @@
 TIMEOUT_MAKE_UPLOAD_NOTEBOOKS = TIMEOUT_MAKE_DOWNLOAD_NOTEBOOKS = 5
 TIMEOUT_MAKE_CLEAN_NOTEBOOKS = 5
 
-TIMEOUT_NEURO_LOGIN = 5
+TIMEOUT_NEURO_LOGIN = 15
```

Comment:
also, until our timeouts are stabilized, we'll adjust them many times in future PRs